Geometry/CaloTopology/src/HcalTopology.cc

```diff
@@ -144,7 +145,7 @@ HcalTopology::HcalTopology(HcalTopologyMode::Mode mode, int maxDepthHB, int maxD
   firstHETripleDepthRing_((mode==HcalTopologyMode::H2 || mode==HcalTopologyMode::H2HE)?(24):(27)),
   singlePhiBins_(72), doublePhiBins_(36),
```

Comment:
@bsunanda - why singlePhiBins_ initialisation is kept unchanged here, while it was changed in the other constructor? BTW, the variable is not modified anywhere. Is it needed as a variable or the IPHI_MAX constant can be used instead?